cabal-install/main/Main.hs

```diff
@@ -1246,3 +1292,23 @@ manpageAction commands flagVerbosity extraArgs _ = do
                  then dropExtension pname
                  else pname
   putStrLn $ manpage cabalCmd commands
+
+--Further commands to support ghc-mod usage
```

Comment:
ghc-mod reference again. If anything it should mention cabal-helper, but I'm not sure that's appropriate.